### ISSUE TITLE
Highcharts Feature Request: Automagically / proportionately resize marker radius on redraw/reflow

### ISSUE THREAD

**Issue by Kinsbane** [highcharts/highcharts#1587]
**TL;DR:** Highcharts does not seem to have built-in support for automatically sizing marker points when resizing a chart; eg, on a mobile phone with viewport width of 320px, a radius of size 10 means the markers overlap and thus are not easily clickable; please see this Fiddle for workaround: http://jsfiddle.net/Kinsbane/w8yTM/3/

I have spent the entirety of my work day today trying to figure out how best to proportionately size my markers in a scatter-based chart based on the chartWidth. I was more than a little dismayed to learn that no such functionality exists within Highcharts, and had to craft my own solution, which works, but with a bug.

Please see this Fiddle: http://jsfiddle.net/Kinsbane/w8yTM/3/

The problem with the Fiddle above is that it does not automatically change the size of the markers after resizing the window - I have to resize two or three times to get the markers to resize, and then, it doesn't seem to be the correct radius for the current size of the chart.

Previously, I hadn't set the Point.update redraw flag to false, and I was getting this error in the console:
Uncaught TypeError: Property 'redraw' of object #<Object> is not a function 

The API documentation for Point.update told me about redrawing the chart after all changes to the points have been made, but, this is a little self-defeating, because if I am redrawing the Points with a new radius within chartOptions.chart.redraw(), I would still get the error above, eg calling chart.redraw() within chartOptions.chart.redraw()

Maybe there is an easier, better way to do this within Highcharts that I'm completely missing after 10 hours of working on this, but I do not think so. Most Google searches and StackOverflows dealing with "dynamically changing the marker" deal with colors only - and nothing about setting the size of the marker radius proportionately based on the size of the chart itself.
